src/components/il-tabs/readme.md

```diff
@@ -0,0 +1,64 @@
+# il-tabs
+
+This is a component that groups a lot of related, parallel text into a collapsable section that is marked with headings where you only want one section visible at a time. This is useful if you expect the user to be scanning for specific information, you have a lot of information on the page, and users will only care about one of the items in the tab section. 
+
+In a mobile view, this is collapsed to a list of tabs with a section under it where the text resides. 
+
+If you only have one block of text that you are hiding or want a traditional accordion, consider using the `<detail>` and `<summary>` tag. Remember that the `<summary>` is considered a button, so any headings you add in the summary tag are ignored. 
```

Comment:
This seems unnecessary

Reply:
Removed. We still are going to have the <details> <summary> still styled though, right?

Reply:
We're not styling anything that's not a component or inside a component anymore